api/turing/config/config.go

```diff
@@ -134,6 +135,17 @@ type EnsemblingJobConfig struct {
 	KanikoConfig KanikoConfig `validate:"required"`
 	// ImageBuilderConfig contains the configuration related to the built ensembler image itself.
 	ImageBuilderConfig ImageBuilderConfig `validate:"required"`
+	// DefaultConfigurations contains the default configurations applied to the ensembling job.
+	// The user is free to override/append to the configuration
```

Comment:
> The user is free to override/append to the configuration

By append/override, do we mean the user-supplied config values on app startup or some other way such as configs specified during job submission? I imagine it's the latter and if so, can it be clarified with a line or two in the comments?